superduperdb/backends/ibis/db_helper.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+
+
+def _default_insert_processor(table_name, datas):
+    """Default insert processor for SQL dialects."""
+    return table_name, datas
+
+
+def _clickhouse_insert_processor(table_name, datas):
+    """Insert processor for ClickHouse."""
+    return f'`{table_name}`', pd.DataFrame(datas)
```

Comment:
Does this do something? `f'\`{table_name}\`'`

Reply:
Some table names we defined can not insert data sometimes, for example, table name with `-`



`_outputs _a-b-c_0`. 

After adding this, the generated sql statement will be "INSERT \`_outputs _a-b-c_0\` xxxxx" in the query of the table name.

Reply:
Ok